lib/clamav/client.rb

```diff
@@ -49,5 +49,18 @@ def resolve_default_socket
         ::UNIXSocket.new(unix_socket || '/var/run/clamav/clamd.ctl')
       end
     end
+
+    def ping
+      execute Commands::PingCommand.new
+    end
+
+    def scan(file)
```

Comment:
I believe this should be marked as private as the only method part of the API seems to be `safe?`